packages/client/Atmosphere.ts

```diff
@@ -115,16 +134,19 @@ export default class Atmosphere extends Environment {
     })
   }
 
-  fetchHTTP = async (body: object, connectionId?: string) => {
+  fetchHTTP = async (body: FetchHTTPData, connectionId?: string) => {
+    const uploadables = body.payload.uploadables
+    delete body.payload.uploadables
```

Comment:
nit: let's fully encapsulate uploadables into a single function. e.g.

```ts
body.payload.uploadables ? toFormData(data) : JSON.stringify(body)
```

that way everything that has to do with uploadables is all done in 1 function that's inside a conditional

Reply:
that's a good idea, changed!